### PR TITLE
Updated for the publishing feature limitation

### PR DESCRIPTION
**Pull request by Stealth7777**
Added additional limitations and exclusions. Please review the [!IMPORTANT] tag as it does not seem to be working

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Invoke-SPOSiteSwap.md [hunk 1]
```diff
@@ -47,7 +47,10 @@ If the target is the root site at <https://tenant-name.sharepoint.com,> then the
 3. Larger tenants that have more than ~10,000 licenses may also need to run the [Page Diagnostic Tool](https://docs.microsoft.com/office365/enterprise/page-diagnostics-for-spo) against the source site. Any analysis results that have the category Attention required (Red) or Improvement opprtunities (Orange) will need to be remediated before performing the swap.
 
 The source and target sites can't be connected to an Office 365 group. They also can't be hub sites or associated with a hub.
-If a site is a hub site, unregister it as a hub site, swap the root site, and then register the site as a hub site. If a site is associated with a hub, disassociate the site, swap the root site, and then reassociate the site.
+If a site is a hub site, unregister it as a hub site, swap the root site, and then register the site as a hub site. If a site is associated with a hub, disassociate the site, swap the root site, and then reassociate the site. 
+
+> [!IMPORTANT]
```
Comment (tseward) [inline sharepoint/sharepoint-ps/sharepoint-online/Invoke-SPOSiteSwap.md:52]: Tags such as !IMPORTANT should not be used in cmdlet documentation as these do not render in PowerShell.

File: sharepoint/sharepoint-ps/sharepoint-online/Invoke-SPOSiteSwap.md [hunk 2]
```diff
@@ -166,7 +169,7 @@ Accept wildcard characters: False
 
 Overrides and ignores any warnings that have been identified by the Page Diagnostic Tool that are preventing a swap from being initiated.
 
-For example, if you receive the warning "Invoke-SPOSiteSwap : Errors: 0; Warnings: 1; Details: The site is not a communications site or it has the classic publishing feature enabled." then this warning can be overriden and ignored using this parameter.
+For example, if you receive the warning "Invoke-SPOSiteSwap : Errors: 0; Warnings: 1; Details: 2 items are not in CDN, please enable CDN." Warnings can be overriden using this parameter but the items should be reviewed and resolved for long-term stability and performance.
```
Comment (tseward) [inline sharepoint/sharepoint-ps/sharepoint-online/Invoke-SPOSiteSwap.md:172]: Is the original warning message not valid? Has it been replaced by the new message?

**Review by tseward — CHANGES_REQUESTED** [review]

**Comment by tseward** [comment]
I would consider rephrasing the messaging on the Publishing feature. Something like:

"If the Publishing feature has previously been or is enabled on the source site, Invoke-SPOSiteSwap cannot be used.".

**Comment by get-itips** [comment]
Agree w/ @tseward suggestions

**Review by get-itips — COMMENTED** [review]
Agree w/ @tseward suggestions

File: sharepoint/sharepoint-ps/sharepoint-online/Invoke-SPOSiteSwap.md (hunk 2 above)
Comment (Stealth7777) [reply sharepoint/sharepoint-ps/sharepoint-online/Invoke-SPOSiteSwap.md:172]: Yes, a classic site is not supported for site swap and should never have been used in the documentation. We only support swapping a modern site to the root. If you look at the site swap documentation you will see that it has been updated as well

File: sharepoint/sharepoint-ps/sharepoint-online/Invoke-SPOSiteSwap.md (hunk 1 above)
Comment (Stealth7777) [reply sharepoint/sharepoint-ps/sharepoint-online/Invoke-SPOSiteSwap.md:52]: Ok

**Comment by JoanneHendrickson** [comment]
@yogkumgit  @tseward @get-itips @Stealth7777     Hi everyone:  This update needs to get out asap as it is causing support issues.  
These updates need to be published before we have another issue because customers did the wrong thing

**Comment by tseward** [comment]
@JoanneHendrickson at the very least, the !IMPORTANT markdown needs to be removed as that doesn't render in PowerShell.

**Comment by Stealth7777** [comment]
Happy to have it removed but don’t see how to remove it now without resubmitting it from scratch?

**Comment by get-itips** [comment]
> Happy to have it removed but don’t see how to remove it now without resubmitting it from scratch? 

One way is:

Click on **Files Changed**, and you can edit the the file from here, for example

image

**Comment by Stealth7777** [comment]
Thanks, done

**Review by tseward — APPROVED** [review]

**Review by JoanneHendrickson — APPROVED** [review]

**Pull request merged**